common/src/main/resources/assets/wynntils/lang/en_us.json

```diff
@@ -658,8 +662,12 @@
   "function.wynntils.horse_tier.name": "Horse Tier",
   "function.wynntils.horse_xp.description": "Your horse's current xp",
   "function.wynntils.horse_xp.name": "Horse XP",
+  "function.wynntils.inventoryFree.description": "Number of open slots in inventory",
+  "function.wynntils.inventoryFree.name": "Open Inventory Slots",
   "function.wynntils.level.description": "Your current combat level",
   "function.wynntils.level.name": "Lvl",
+  "function.wynntils.liquidEmerald.description": "Amount of liquid emeralds in inventory",
```

Comment:
Amount of money in liquid emeralds in your inventory

Currently it sounds like this function counts LE items, not overall money

Also make sure to reword eb and em translations